earth2studio/data/xr.py

```diff
@@ -169,3 +169,114 @@ def __call__(
             arrs.append(self.das[yr][mon].sel(time=tt, variable=variable))
 
         return xr.concat(arrs, dim="time")
+
+
+class DataArrayPathList:
+    """A local xarray dataarray directory data source that handles multiple files.
```

Comment:
Add this to the docs here to the API shows up

https://github.com/NVIDIA/earth2studio/blob/main/docs/modules/datasources.rst?plain=1#L39